tasks/noci_test.sh

```diff
@@ -10,7 +10,4 @@ npm run test-jasmine -- --tags=noCI --nowatch || EXIT_STATE=$?
 # mapbox image tests take too much resources on CI
 npm run test-image -- mapbox_* --queue || EXIT_STATE=$?
 
-# run gl2d image test again (some mocks are skipped on CI)
-npm run test-image-gl2d || EXIT_STATE=$?
```

Comment:
fixup for https://github.com/plotly/plotly.js/pull/2258/commits/9f304e658bfd8cecd6a5a011faa46e6161deaa58 

No need for this line anymore as now all gl2d mocks are tested on CI.